coding-concepts.js

```diff
@@ -67,7 +76,7 @@ class Learn {
   }
 }
 var learnStudent = new Learn("George")
-// console.log(learnStudent)
+console.log(learnStudent)
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Im not too sure about this one because when doing the lecture there wasn't enough visuals/examples for me to understand well enough. I did go over the notes provided through the syllabus but the coding looks weird to me. What I know is that this is a dynamic class that has some information. Maybe it will input the string George into line 64 where name is since everthing else is dead code. Cement code. I forgot the word.
+// b) Verify and explain: Okay so when we did invoke it I was right. What's happening here we have a class and object. i don't want to describe any of them because I'm afraid that I would use class and object interchangebly. There is something that I comepletely missed when initially looking at the code. At line 63 the constructor method is taking an argument of name. The input of name is arbutrary because it could be anything that is a string. If I had saw and take notice of that I don't think I would have been so confused. The code on line 69 is create a new instance of class and we know because of the key word new but right next to the word we have the class Learn. We basically plugged in the string George and passed it as an argument.
```

Comment:
Awesome!  Classes and objects are kind of tricky to distinguish at first.  You can think of classes as an object maker.  An object can have one or many instances of key: value pairs, and the class is a template to create new objects.